lib/util/import-target.js

```diff
@@ -19,10 +21,19 @@ function getFilePath(isModule, id, options) {
     try {
         return resolve.sync(id, options)
     } catch (_err) {
-        if (isModule) {
-            return null
+        try {
+            const {url} = importResolve(id, {
+                    parentURL: pathToFileURL(path.join(options.basedir, 'dummy-file.js')).href,
+                    conditions: ['node', 'import', 'require']
```

Comment:
why the `conditions` is needed?

Reply:
Because the default `conditions` (in the `import-meta-resolve` package) is `['node', 'import']`, so will not resolve CJS. But there are packages out there, that are CJS packages, that use `exports` and not `main`. So I added the `require` condition to support them too.